documentation/book/con-certificates.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='certificates-{context}']
+= Certificates
+
+Each {ProductName} component needs its own private keys and public key certificates in order to support encryption.
+All component certificates are signed by a Certification Authority (CA) called _cluster CA_.
```

Comment:
All component certificates are signed by a Certification Authority (CA) called _cluster CA_.
A separate CA, the  _clients CA_, is used to signed the certificates of Kafka clients.

Change to:

All component certificates are signed by a Certification Authority (CA) called _cluster CA_, while the _clients CA_, is used to sign the certificates for the Kafka clients.

Reply:
Is the comma after "clients CA" necessary? It makes it read funny imho.